pypeclub/OpenPype
# nuke: fixing opening loader with context if shortcut is used

## Brief description
Nuke is opening loader with context now.

## Description
Expected behaviour was open Loader in context wather it is open form menu or with shortcut. It didn't work.

## Testing notes:
1. Open Nuke in any shot context
2. press combination `CTRL` + `ALT` + `L`
3. it should open Loader in context of the shot
